tests/unit/macaroons/test_services.py

```diff
@@ -69,17 +58,12 @@ def test_find_macaroon(self, user_service, macaroon_service):
         assert isinstance(dm, Macaroon)
         assert macaroon.id == dm.id
 
-    def test_find_userid_no_macaroon(self, macaroon_service):
-        assert macaroon_service.find_userid(None) is None
```

Comment:
The MacaroonService.find_userid method was never called with None. I'm not sure it makes sense to leave this codepath.